papers/whitepaper.md

```diff
@@ -0,0 +1,93 @@
+
+In order to empower the next generation of applications in the intersection of Web3 and AI, both data and model privacy is of the utmost importance. Through the use of TEEs, we will not only be able to provide verifiable AI compute, but also provide full data privacy to the end user.
+
+This means that Atoma nodes will not be able to access any sensitive data, such as personal information, while also not being able to access the output generated by the AI model. Only the end user will be able to decrypt the generated output, and can have high assurance that the input to the model was not shared with any third party.
+
+Through this, users can have full control of their data, allowing for AI models to operate on sensitive data. Privacy will unlock the full potential of AI, allowing for the creation of new data centric economic revenue models as well as applications that can operate on the user's behalf without revealing anything about the user.
+
+The Atoma Network can be used by Web2 companies and builders, specially software companies that are built from the ground up to be AI-centric, fast growing AI providers (such as Anthropic, etc) and software companies committed to integrating AI into their products (think of Canvas, etc). The reason for these companies to want to operate on the Atoma Network comes from the fact that a network like Atoma is positioned to drive down the costs of GPU utilization. 
+
+Due to the current GPU chips shortage, together with a waste of GPU utilization (meaning less GPU cycles being used for AI inference), hardware providers face sunk costs by not utilize their infrastructure efficiently. Therefore, such hardware providers are able to offer those fractional GPU cycles at a much lower cost compared to other larger cloud providers. Notice, for example, that renting an H100 from AWS today, customers need to commit to a 1-year lease, due to market supply constraints. It is also unlikely that a customer needs to use a GPU at full capacity during the 1-year period. There is furthermore, a high number of idle GPUs in the market, that were used before as mining hardware (e.g., when ETH transitioned from PoW to PoS). Such hardware available can be repurposed for AI inference.
+
+The development of new AI models will be crucial for the future of the Atoma Network. We believe that open source AI development will be crucial to democratize and broaden the reach of AI technologies across societies. We are committed to incentivize the open source AI community to develop and deploy new AI models, while being able to monetize their work, allowing for experts in the community to earn a continuous stream of revenue through their model utilization.
```

Comment:
The development of new AI models will be crucial for the future of the Atoma Network. We believe that open source AI development will democratize and broaden the reach of AI technologies across societies. We are committed to incentivize the open source AI community to develop and deploy new AI models, while being able to monetize their work. This allows experts in the community to earn a continuous stream of revenue through their model utilization.